### PR TITLE
Homework exercise navigation page

### PR DESCRIPTION
**Pull request by AnnaDinaburgVulikh**
1. Created a navigation page for homework exercises.
2. Updated the side menu in course materials to modular form using side-menu.js
3. Linked the homework navigation page to the  Rose main page and to the course materials side bar.

You can view the changes in work here: https://annadinaburgvulikh.github.io/ROSE/ 
by navigating using the homework link after materials or navigating to materials and using the side bar.
Please let me know when to remove the temporary page.

### CODE REVIEW

**Comment by AnnaDinaburgVulikh** [comment]
Hi @shiramax, please let me know if any fixing needed.

**Comment by shiramax** [comment]
@sleviim it's already exposed in Github.

**Comment by shiramax** [comment]
@sleviim it's already exposed in Github.
there is no homework at the moment for python intro, we can delete this section for now.

**Comment by sleviim** [comment]
@shiramax, do we want to expose all homework ahead?
@anna, I think there's something missing with the 'python exercise' link

**Review by shiramax — CHANGES_REQUESTED** [review]
@AnnaDinaburgVulikh can you add some tests to make sure the links won't break? 
(In case someone will change the names of the files)

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 1]
```diff
@@ -1,41 +1,38 @@
...
+10. Use the command diff and check the difference between the files **my_song.txt** and **new_song.txt**. (don't forget to use man).
 
- 14. Print the content of the file **your_song.txt**.
+11. Move the file  **new_song.txt** to ~/hw2/drafts.
 
- 15. Clear the terminal screen.
+12. List in a tree format all the directories in your home directory.
 
- 16. Bonus - Use the command **wc** to print how many lines you have in  ~/hw2/songs/your_song.txt (don't forget to use man).
+13. Change the name of **my_song.txt** to **your_song.txt** (The file is in ~/hw2/songs).
```
[the remark this answers is not in the document]
Comment (AnnaDinaburgVulikh) [reply docs/course_materials/exercises/01_Linux/homework_2.md:38]: not intended, I'll copy the latest from the repo

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 2]
```diff
@@ -36,3 +34,9 @@ Please write down your answers, including the command you used and the output yo
 12. List in a tree format all the directories in your home directory.
 
 13. Change the name of **my_song.txt** to **your_song.txt** (The file is in ~/hw2/songs).
+
+14. Print the content of the file **your_song.txt**.
+
+15. Clear the terminal screen.
+
+16. Bonus - Use the command **wc** to print how many lines you have in  ~/hw2/songs/your_song.txt (don't forget to use man).
```
Comment (sleviim) [inline docs/course_materials/exercises/01_Linux/homework_2.md:42]: in  ~/hw2/songs/your_song.txt > remove extra space

**Comment by AnnaDinaburgVulikh** [comment]
@shiramax @sleviim I hid the link to the homework in the menu under number 9. It's in white on white background, if you hover on you'll see the cursor change.

**Review by shiramax — APPROVED** [review]
Great work @AnnaDinaburgVulikh thanks!

**Review by shiramax — APPROVED** [review]

**Pull request merged**